### ISSUE TITLE
how to achieve the acc in the paper

### ISSUE THREAD

**Issue by FANG-MING** [cfzd/Ultra-Fast-Lane-Detection#255]
I had trained the network with backbone is res18,but cannot attach 68.4 f1-score in culane dataset. Coudle you help me? thanks

**Comment by cfzd** [comment]
@FANG-MING 
Are you using the default training parameters? If you are using the default training parameters, just run
```
sh launch_training.sh
```
Then you can get the results. The provided models are trained in exactly the same way.

**Comment by FANG-MING** [comment]
i had tried the single gpu trained, that will have influence?

**Comment by cfzd** [comment]
@FANG-MING 
If you want to train with 1 GPU, you should reduce the learning rate from 0.1 to 0.025.

**Comment by FANG-MING** [comment]
> @FANG-MING What's your device? What're the results of running speed_simple.py?

I test in 1080ti, but got speed is 260fps fastest

**Comment by cfzd** [comment]
@FANG-MING 
it's possible since the latency is already so low. The inference time for 300fps is 3.3ms. For 260fps, it is 3.84ms. Any latency fluctuation could make the speed slower.

**Comment by ma-cg** [comment]
Hello, author, why can't I reach the accuracy of the paper when I reproduce it with the official code of resnet34 network

**Comment by cfzd** [comment]
@ma-cg 
It is possible since the released repo is a re-implementation. We will release a new version very soon, in which the res34 could achieve a ~76 F1 measure.

**Comment by ma-cg** [comment]
When will the new version be released? I've been waiting for a long time
